lib/help.js

```diff
@@ -19,6 +19,7 @@ module.exports = `
       --priority, -p     Update priority of task
       --archive, -a      Display archived items
       --restore, -r      Restore items from archive
+      --renameBoard -r   Rename board; boards with spaces - specify via "_" symbol like My_Board
```

Comment:
I feel like `--rename-board` would be more standard here :)

Reply:
Yes, `--rename-board` is more standard.

Reply:
Also it looks like your short option `-r` conflicts with the restore command